docs/howtoguides/enable_livepatch.md

```diff
@@ -22,13 +42,47 @@ Installing canonical-livepatch snap
 Canonical livepatch enabled.
 ```
 
-To check the status of Livepatch once it has been installed use the following
-command:
+## Check Livepatch status after installation
+To check the status of the Livepatch client once it has been installed use the
+following command:
```

Comment:
We're working to surface the most important information up through `pro status`, so this should be more like
```suggestion
## Check Livepatch status after installation
If you're interested in the detailed status of the Livepatch client once it has been installed use the
following command:
```